plugins/kubernetes/app/decorators/build_decorator.rb

```diff
@@ -0,0 +1,14 @@
+Build.class_eval do
+
+  has_many :kubernetes_release_groups, class_name: 'Kubernetes::ReleaseGroup'
+  attr_writer :project_name
+  attr_accessor :env
+
+  def project_name
+    @project_name ||= project.name.parameterize('-')
```

Comment:
I'd rather see this on project (`parameterized_name` or something), but it's fine